src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+    private static final Lottery lottery1 = new Lottery();
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = lottery1.getRandomBall();
```

Comment:
lottery[i] is an object which is an instance of Lorttery class, so you can call getRandomBall() on it, you don't need lottery1.

Reply:
when i do this way i have an arr "lottery[I] is null" 
my code is  lottery[i] = lottery[I].getRandomBall();

Reply:
fixed